src/Utilities/api.js

```diff
@@ -143,12 +143,26 @@ export const fetchExecutiveReport = async () => {
 
 };
 
-export const fetchSuggestedInstanceTypes = async (params) => {
+export const fetchSuggestedInstanceTypes = async (fetchParams) => {
+    const { page, perPage, activeSortIndex, activeSortDirection } = fetchParams || {};
+    let orderBy = 'system_count';
+
+    if (activeSortIndex === 0) {
+        orderBy = 'instance_type';
```

Comment:
@r14chandra [similar to comment here](https://github.com/RedHatInsights/ros-frontend/pull/292/files#r1395531923) we can add a key in the sugg_instance_column for order_by value and accept it in the fetchParams? thougths?